docs/LCIAmethods.md

```diff
@@ -0,0 +1,42 @@
+# Life Cycle Impact Assessment (LCIA) Methods
+
+A number of LCIA methods have been created for use with the Federal Elementary Flow List (FEDEFL) and the Federal LCA Commons (FLCAC).
+The [LCIA Formatter](https://github.com/USEPA/LCIAformatter) is the primary tool for creating these methods.
+
+> LCIA formatter, or lciafmt, is a Python tool for standardizing the format and flows of life cycle impact assessment (LCIA) data.
+> The tool acquires LCIA data transparently from its original source, cleans the data, shapes them into a standard format using the LCIAmethod format, and optionally applies flow mappings as defined in the FEDEFL.
+
+An article describing the [LCIA Formatter was published in the Journal of Open Source Software (JOSS)](https://doi.org/10.21105/joss.03392)
+
+## Available Methods
+
+|LCIA Data|Provider|Link|
+|---|---|---|
+|TRACI 2.1|US Environmental Protection Agency|[Tool for Reduction and Assessment of Chemicals and Other Environmental Impacts](https://www.epa.gov/chemical-research/tool-reduction-and-assessment-chemicals-and-other-environmental-impacts-traci)|
+|ReCiPe 2016 Midpoint|National Institute for Public Health and the Environment (The Netherlands)|[LCIA: the ReCiPe Model](https://www.rivm.nl/en/life-cycle-assessment-lca/recipe)|
+|ReCiPe 2016 Endpoint|National Institute for Public Health and the Environment (The Netherlands)|[LCIA: the ReCiPe Model](https://www.rivm.nl/en/life-cycle-assessment-lca/recipe)|
+|ImpactWorld+ Midpoint*|International Reference Center for Life Cycle of Products, Services and Systems (CIRAIG)|[ImpactWorld+](http://www.impactworldplus.org/en/team.php)|
```

Comment:
What does the * indicate? I expected a table note.

Reply:
thanks - copied over from somewhere else, not needed here.